tests/model/schema.js

```diff
@@ -1123,12 +1123,12 @@ describe( 'Schema', () => {
 			schema.extend( 'img', { allowAttributes: 'bold' } );
 			schema.extend( '$text', { allowIn: 'img' } );
 
-			setData( model, '[<p>foo<img>xxx</img>bar</p>]' );
```

Comment:
This changes the semantics of this test. Let's better use `parse()`, especially that later on we do `selection.getRanges()` anyway.

Reply:
This is still not resolved.